contracts/mock/WhitelistMock.sol

```diff
@@ -0,0 +1,17 @@
+pragma solidity ^0.4.24;
+
+import "../LinniaWhitelistI.sol";
+
```

Comment:
It is a lot cleaner if you only implement the `expertScoreOf`

Reply:
how are they suppose to add the users and score?

Reply:
for mock/tests I think the updateScore is needed